app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -294,6 +295,7 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
             // Exit PiP mode
             moveTaskToBack(false)
             startActivity(Intent(this, this::class.java).setFlags(FLAG_ACTIVITY_REORDER_TO_FRONT))
+            exitImmersiveModeIfNeeded()
```

Comment:
Immersive mode just means the system widgets - toolbar / navigation bar are hidden and the app will extend to cover the entire screen.
Exiting immersive mode will mean that the status bar / navigation bar can be shown again.
Looking at the video showcasing the issue (from https://github.com/mozilla-mobile/fenix/issues/22471#issue-1055723461) I see the status so I'm not sure exiting immersive mode here is needed to fix the issue.

Reply:
Noted. Will search for another fix.

Reply:
Noted. Will search for another fix.
Apologies for the late response.

Reply:
Noted. Will search for another fix.
Apologies for the late response.